Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,33 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function GryffindorHouse(obj){
+  let num = 1;
+  for(let index in hogwarts){
+    let {firstName} = hogwarts[index];
+    let {house} = hogwarts[index];
+    let {lastName} = hogwarts[index];
+    if(house == 'Gryffindor'){
+      console.log(`${num}-` + firstName, lastName);
+      num += 1;
```

Comment:
Nice touch with the numbering, is there another way you could achieve that without having to use the num variable?